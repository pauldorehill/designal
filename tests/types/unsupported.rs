use designal::Designal;

#[derive(Designal)]
enum HumanBean {}

#[derive(Designal)]
union HumanBean1 {
    taste: u8
}

#[derive(Designal)]
struct HumanBean2;

fn main() {}