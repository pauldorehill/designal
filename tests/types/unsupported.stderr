error: Enums are not yet supported
 --> $DIR/unsupported.rs:4:6
  |
4 | enum HumanBean {}
  |      ^^^^^^^^^

error: Unions are not yet supported
 --> $DIR/unsupported.rs:7:7
  |
7 | union HumanBean1 {
  |       ^^^^^^^^^^

error: Unit structs are not supported
  --> $DIR/unsupported.rs:12:8
   |
12 | struct HumanBean2;
   |        ^^^^^^^^^^
